http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/message/HTTPMessageUtil.java

```diff
@@ -131,14 +131,18 @@ private static CarbonMessage setWebSocketCommonProperties(CarbonMessage cMsg,
      * Extract sender configuration from transport configuration.
      *
      * @param transportsConfiguration {@link TransportsConfiguration} which sender configurations should be extracted.
+     * @param scheme
      * @return extracted {@link SenderConfiguration}.
      */
-    public static SenderConfiguration getSenderConfiguration(TransportsConfiguration transportsConfiguration) {
+    public static SenderConfiguration getSenderConfiguration(TransportsConfiguration transportsConfiguration,
```

Comment:
This method does not fit to the intention of this class (HTTPMessageUtil). Shall we move these transport configuration related methods to a separate utility class?

Reply:
What if we change the class name to something broader? i.e: HTTPUtils?